python/packages/ai/teams/ai/action/action_entry.py

```diff
@@ -0,0 +1,27 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from typing import Any, Awaitable, Callable, Generic, TypeVar, Union
+
+from botbuilder.core import TurnContext
+
+from teams.ai.turn_state import TurnState
+
+StateT = TypeVar("StateT", bound=TurnState)
+ReturnT = TypeVar("ReturnT")
+ActionFunctionSync = Callable[[TurnContext, TurnState, Any, str], Union[None, bool]]
```

Comment:
JS SDK requires the action to return a bool. Is there any reason we want to allow function return `None` here?

Reply:
the C# implementation supports void https://github.com/microsoft/teams-ai/blob/main/dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Action/ActionHandler.cs#L28 and I think it makes sense to support and default to false

Reply:
Can you create an issue so Corina can follow up in JS SDK (and can discuss there if there's difference opinion) ? I'm neutral to this and my only goal is keeping the functionality aligned across all 3 SDKs.